entrypoint.sh

```diff
@@ -44,6 +44,7 @@ elif [ -n "$POSTGRES_PORT_5432_TCP" ]; then
 		echo >&2 "  Connecting to DB_HOST ($DB_HOST)"
 		echo >&2 "  instead of the linked postgres container."
 	fi
+	: ${DB_NAME:='postgres'}
```

Comment:
arguably we should detect those alternative database environment variables, and use those? E.g.

```
: ${DB_NAME:=$MYSQL_ENV_MYSQL_DATABASE}
: ${DB_NAME:="drupal"}
```

and

```
: ${DB_NAME:=$POSTGRES_ENV_POSTGRES_DB}
: ${DB_NAME:="postgres"}
```